policy/pa_test.go

```diff
@@ -504,23 +505,23 @@ func TestCheckAuthzChallenges(t *testing.T) {
 		{
 			name: "no challenges",
 			authz: core.Authorization{
-				Identifier: identifier.ACMEIdentifier{Type: identifier.DNS, Value: "example.com"},
+				Identifier: identifier.ACMEIdentifier{Type: identifier.TypeDNS, Value: "example.com"},
```

Comment:
Why doesn't this use `identifier.NewDNS`?

Reply:
Ah, these got missed because I created the "clean up use of identifier package" commit _before_ rebasing on top of https://github.com/letsencrypt/boulder/pull/7677 which introduced these test cases. Good catch.